README.md

```diff
@@ -328,77 +294,66 @@ To enable multiple reporters, simply define multiple reporters with all the nece
 for each one.  For example, to report to chef-compliance and write to json file on disk:
 
 ```ruby
-"audit": {
-  "reporter": [ "chef-compliance", "json-file" ]
-  "server": "https://compliance-fqdn/api",
-  "owner": "my-comp-org",
-  "refresh_token": "5/4T...g==",
-  "insecure": false,
-  "profiles": [
-    {
-      "name": "windows",
-      "compliance": "base/windows"
-    }
-  ]
-}
+default['audit']['reporter'] = ['chef-server-automate', 'json-file']
+default['audit']['profiles'].push(
+  {
+    'name': 'windows',
+    'compliance': 'base/windows'
+  }
+)
 ```
 
 ### Profile Fetcher
 
 #### Fetch profiles from Chef Automate/Chef Compliance via Chef Server
 
 To enable reporting to Chef Automate with profiles from Chef Compliance or Chef Automate, you need to have Chef Server integrated with [Chef Compliance or Chef Automate](https://docs.chef.io/integrate_compliance_chef_automate.html#collector-chef-server-automate). You can then set the `fetcher` attribute to 'chef-server'.
+
 This will allow the audit cookbook to fetch profiles stored in Chef Compliance. For example:
 
 ```ruby
-"audit": {
-  "fetcher": "chef-server",
-  "reporter": "chef-server-automate",
-  "profiles": [
-    {
-      "name": "ssh",
-      "compliance": "base/ssh"
-    }
-  ]
-}
+default['audit']['reporter'] = 'chef-server-automate'
+default['audit']['fetcher'] = 'chef-server'
+default['audit']['profiles'].push(
+  {
+    'name': 'ssh',
+    'compliance': 'base/ssh'
+  }
+)
 ```
 
 #### Fetch profiles directly from Chef Automate
 
-This method is fetching profiles using the `data_collector.server_url` and `data_collector.token`, defined in `client.rb`. It requires `inspec` version `0.27.1` or greater. Further information is available at Chef Docs: [Configure a Data Collector token in Chef Automate](https://docs.chef.io/ingest_data_chef_automate.html)
+This method is fetches profiles using the `data_collector.server_url` and `data_collector.token`, defined in `client.rb`. It requires `inspec` version `0.27.1` or greater. Further information is available at Chef Docs: [Configure a Data Collector token in Chef Automate](https://docs.chef.io/ingest_data_chef_automate.html)
```

Comment:
s/is fetches/fetches 
s/using the/using
s/, defined/, which are (also in similar example above)